### ISSUE TITLE
It should be possible to configure key server and home did for rvm gpg_key

### ISSUE THREAD

**Issue by lesniakania** [sous-chefs/chef-rvm#299]
I'm using `vagrant` with the newest Ubuntu (ubuntu/trusty64) and I have an issue with hkp://keys.gnupg.net being blocked by firewall. I would like to be able to configure key server in my cookbook.

Also there is an issue if you run cookbook as `vagrant` user - signature is saved in the `/home/vagrant/.gnupg` directory and when later you try install `rvm::system` cookbook tries to find signature in `/root/.gnupg` so installation fails. It would be great to be able to configure both key server and home dir then.

Actually I've already did that and I'll make pull request in a minute, I just wanted to be consistent and add issue too.

**Comment by damacus** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs
